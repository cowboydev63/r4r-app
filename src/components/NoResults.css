.results__noresults {
    margin: 0px auto 40px auto;
    font-style: italic;
}