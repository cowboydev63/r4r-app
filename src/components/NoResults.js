import React from 'react';
import PropTypes from 'prop-types';
import { Theme } from '../theme';
import './NoResults.css';

const NoResults = ({ viewAll }) => (
    <Theme element="div" className="results__noresults">No results were found for your search. Please try another search or <a onClick={ viewAll }>view all resources.</a></Theme>
)

NoResults.propTypes = {
    viewAll: PropTypes.func.isRequired,
}

NoResults.defaultProps = {
    viewAll: () => {}
}

export default NoResults;