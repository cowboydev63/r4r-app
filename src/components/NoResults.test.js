import React from 'react';
import { shallow } from 'enzyme';
import NoResults from './NoResults';

describe('NoResults Component', () => {

  it('renders correctly', () => {
    const wrapper = shallow(<NoResults/>);
    expect(wrapper).toMatchSnapshot();
  })
})