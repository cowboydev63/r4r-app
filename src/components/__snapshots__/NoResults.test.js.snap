// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults Component renders correctly 1`] = `
<Theme
  className="results__noresults"
  element="div"
>
  No results were found for your search. Please try another search or 
  <a
    onClick={[Function]}
  >
    view all resources.
  </a>
</Theme>
`;
